LFPy/cell.py

```diff
@@ -924,6 +925,69 @@ def get_rand_idx_area_and_distribution_norm(self, section='allsec', nidx=1,
             p /= p.sum()
             return alias_method(poss_idx, p, nidx)
 
+    def enable_extracellular_stimulation(self, electrode, t_ext=None, n=1, seed=None):
+        """
+        Enable extracellular stimulation with 'extracellular' mechanism.
```

Comment:
I think here it would be important to describe the assumptions of the in silico extracellular stimulation procedure.